maven/await-artifact/action.yml

```diff
@@ -0,0 +1,29 @@
+name: maven/await-artifact
+description: |
+  Waits for an artifact to be available on maven central
+inputs:
+  artifact-id:
+    description: 'Maven artifact-ID of the artifact'
+    required: true
+  group-id:
+    description: 'Maven group-ID of the artifact'
+    required: true
+  version:
+    description: 'Version of the artifact to wait for'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Wait for artifact to be available on maven central
```

Comment:
I don't see a timeout mechanism, but I understand you can set the timeout at the workflow level.

Do you think we should at least mention it in the README?

Reply:
> Do you think we should at least mention it in the README?

Sure thing, I didn't want to improve what it was already implemented as part of the migration

Reply:
Just a note for future improvements:

I remember you implemented an exponential backoff retry somewhere.. I guess this can also benefit from it.

Reply:
see [`b15b936` (#118)](https://github.com/elastic/oblt-actions/pull/118/commits/b15b936aae9e827ae90bbc29d442c267d3ba375a)